src/object_properties.h

```diff
@@ -54,6 +54,7 @@ struct ObjectProperties
 	std::string infotext;
 	//! For dropped items, this contains item information.
 	std::string wield_item;
+	bool no_static_save = false;
```

Comment:
the benefit is in code we use positive meaning to have a comprehensible logic and prevent double negation